tracker/status.go

```diff
@@ -14,12 +14,21 @@ type DeployMessage struct {
 	Retries int
 }
 
+//PodInfo holds information about a Pod
+type PodInfo struct {
```

Comment:
can we get pod's image version? i.e. git SHA tags

Reply:
Would it be fine to just have a list of the images from the containers in the pod? Otherwise is there a consistent way to know which container(s) are of interest in a given pod